NitroxClient/Communication/Packets/Processors/CraftingProcessor.cs

```diff
@@ -0,0 +1,85 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.Unity.Helper;
+using NitroxModel.Core;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class CrafterStartUseProcessor : ClientPacketProcessor<CrafterStartUse>
+    {
+        private readonly IPacketSender packetSender;
+
+        public CrafterStartUseProcessor(IPacketSender packetSender)
+        {
+            this.packetSender = packetSender;
+        }
+
+        public override void Process(CrafterStartUse packet)
+        {
+            GameLogic.Crafting.Instance.GhostCrafter_Remote_OnStartUse(packet.CrafterGuid, packet.PlayerName);
+        }
+    }
+
+    public class CrafterEndUseProcessor : ClientPacketProcessor<CrafterEndUse>
```

Comment:
Our pattern is a single file per packet type.  This allows users to easily jump to the file they need to edit without hunting through various files.

Reply:
Not to have to jump through files was the intended goal. All processors that are related to the Crafting-Logic are concentrated here. 
As already mentioned before the point of view for this architecture is:

[Pach-Classes] >> CallMethods > [Logic-Class with the magic (e.g. Crafting)] < CallMethods << [Processors (e.g. CraftingProcessor)]